packages/optimize/android/build.gradle

```diff
@@ -0,0 +1,39 @@
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.1'
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 31
+//    buildToolsVersion "23.0.1"
+
+    defaultConfig {
+        minSdkVersion 19
+        targetSdkVersion 31
+        versionCode 1
+        versionName "1.0"
+    }
+    lintOptions {
+        abortOnError false
+    }
+}
+
+repositories {
+    google()
+    mavenCentral()
+    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }   
+}
+
+dependencies {
+    api "com.adobe.marketing.mobile:optimize:1.0.0-SNAPSHOT"
```

Comment:
Have we released optimize to production?

Reply:
Yup its publicly available now

Reply:
Then we can use the link without Snapshot.
api "com.adobe.marketing.mobile:optimize:1.+"